recipe/meta.yaml

```diff
@@ -1,29 +1,16 @@
-{% set version = "1.9.6" %}
+{% set version = "2.0.0" %}
 
 package:
   name: pyproj
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pyproj/pyproj-{{ version }}.tar.gz
-  sha256: e0c02b1554b20c710d16d673817b2a89ff94738b0b537aead8ecb2edc4c4487b
-  patches:
-    - data_dir.patch
+  sha256: aa8f0fda988353f37831356d82aadafa4b44815d3c4daadcda046f0f345567ce
 
 build:
-  number: 1000
+  number: 0
   script:
-    - del _proj.c  # [win]
-    - rmdir src  # [win]
-    - set "PROJ_DIR=%PREFIX%"  # [win]
-    - set "PROJ_LIBDIR=%LIBRARY_LIB%"  # [win]
-    - set "PROJ_INCDIR=%LIBRARY_INC%"  # [win]
-    - rm -d _proj.c  # [not win]
-    - rm -rf src
-    - export PROJ_DIR="${PREFIX}"  # [not win]
-    - export PROJ_LIBDIR="${PREFIX}/lib"  # [not win]
-    - export PROJ_INCDIR="${PREFIX}/include"  # [not win]
-    - cythonize -a -i _proj.pyx
```

Comment:
The re-cythonization is desirable but the code base changed a lot and we probably need a better strategy than this one.

Reply:
Sounds good. Feel free to tinker with it 👍

Reply:
I'll check the code as soon as possible but I'm probably not going to be able to get to this until next week.

Reply:
Note: the .c files were removed in the new version.

Reply:
Awesome! One less build step for us :smile:

Reply:
@snowman2 if you are in a hurry to get this merge just add yourself as a maintainer, skip Python 2.7 on Windows, and open an issue about the cythonization (so I can remember to address it later).